api/src/test/java/fade/mirror/MirrorTest.java

```diff
@@ -51,7 +53,7 @@ void testMirrorOfClassHasFields() {
     @DisplayName("mirror of class has correct number of methods")
     void testMirrorOfClassHasCorrectNumberOfMethods() {
         MClass<MockClass> mockClass = mirror(MockClass.class);
-        assertEquals(6, mockClass.getMethodCount(), "'mockClass.getMethodCount()' should return '6'");
+        assertEquals(7, mockClass.getMethodCount(), "'mockClass.getMethodCount()' should return '6'");
```

Comment:
```suggestion
        assertEquals(7, mockClass.getMethodCount(), "'mockClass.getMethodCount()' should return '7'");
``` The error string should change as well :)